playwright/tests/pageobjects/DataClassEditor.ts

```diff
@@ -6,7 +6,8 @@ import { Settings } from './Settings';
 import { Button } from './abstract/Button';
 import { Table } from './abstract/Table';
 
-const server = process.env.BASE_URL ?? 'http://localhost:8081';
+export const server = process.env.BASE_URL ?? 'http://localhost:8081';
+export const user = 'Developer';

```

Comment:
I would introduce a global user constant

Reply:
This is a global constant? :)

Reply:
good for me :D